css/project.less

```diff
@@ -21,27 +21,31 @@
     transition: width .3s ease-in-out .3s, border-width .3s ease-in-out .3s, padding .3s linear .3s;
   }
 
+  &._resize {
```

Comment:
We don't typically use _ prefixes on classes or variables in Caret (the sandboxing means we can't have real public/private API separation, so I'm pretty cavalier about it). You can just call this `resizing` instead.